build/yaml/crd/nsx.vmware.com_nsxserviceaccounts.yaml

```diff
@@ -45,11 +45,80 @@ spec:
                 type: string
               clusterName:
                 type: string
+              conditions:
+                items:
+                  description: "Condition contains details for one aspect of the current
```

Comment:
This description seems too general and it's for developer. Is it a way to change it?

Reply:
It's generated from the Condition type, and I can remove it manually. Not sure how to make kubebuilder ignore the comments of Condition when generating yaml.